lbr_bringup/scripts/lbr_state_smoothing_node.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+import rclpy
+from rclpy.node import Node
+from lbr_state_msgs.msg import LBRState
+
+import numpy as np
+
+
+class LBRStateSmoothingNode(Node):
+
+
+    def __init__(self):
+
+        super().__init__('lbr_state_smoothing_node')
+
+        # Declare and get parameters
+        self.declare_parameter('max_window_length', 20)
+        self._max_window_length = int(self.get_parameter('max_window_length').value)
+        self._window = []
+
+        # Set filter method
+        self._filter = self._filter_mean
+        # self._filter = self._filter_median
+
+        # Create publishers/subscribers
+        self._lbr_state_smooth_publisher = self.create_publisher(LBRState, "/lbr_state/smooth", 1)
```

Comment:
NIT: subscriber first, then publisher (logical order)

Reply:
disagree: the subscriber can be called before the publisher is created - I've had issues with this in past (maybe its a solved issue in ros2)